src/Pages/VideoPlayer/VideoPlayer.jsx

```diff
@@ -0,0 +1,170 @@
+    return response.data.success;
+  };
+
+  useEffect(() => {
+    currentUser &&
+      (async function () {
+        if (
+          !playlists
+            .find((list) => list.id === "history")
+            .videos.find((video) => video.videoId === videoId)
+        ) {
+          const response = await updatePlaylist(
+            currentUser.userId,
+            "history",
+            video._id
+          );
+          response
+            ? dispatch({
+                type: "ADD_TO_PLAYLIST",
+                payload: { playlistId: "history", video: video },
+              })
+            : console.error("Error while updating in DB");
+        }
+      })();
+  }, [currentUser, dispatch, playlists, video, videoId]);
+
+  return (
+    <div>
+      <Navbar />
+      <Sidebar />
+      {!data ? (
+        <div className="canvas">
+          <Loader />
+        </div>
+      ) : (
+        <div className="canvas">
+          <div className="video-player">
+            <div className="video">
+              <iframe
+                width="100%"
+                height="100%"
+                src={`https://www.youtube.com/embed/${videoId}`}
+                title={title}
+                frameBorder="0"
+                allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
+                allowFullScreen="allowfullscreen"
+              ></iframe>
+            </div>
+            <h1>{title}</h1>
+            <div className="engagements">
+              <div className="engagements-left">
+                <div>{views} views</div>
+                <div>{likes} likes</div>
+              </div>
+              <div className="engagements-right">
+                {console.log(
+                  "Output of isVideosInPlailist",
+                  isVideoInPlaylist("liked", videoId)
+                )}
+                {isVideoInPlaylist("liked", videoId) ? (
+                  <ThumbUpIcon
+                    className="engagements-right-selected"
+                    onClick={async () => {
+                      const response = await updatePlaylistInDb("liked", video);
+                      response &&
+                        dispatch({
+                          type: "REMOVE_FROM_PLAYLIST",
+                          payload: { playlistId: "liked", video: video },
+                        });
+                    }}
+                  ></ThumbUpIcon>
+                ) : (
+                  <ThumbUpIcon
+                    onClick={async () => {
+                      const response = await updatePlaylistInDb("liked", video);
+                      response &&
+                        dispatch({
+                          type: "ADD_TO_PLAYLIST",
+                          payload: { playlistId: "liked", video: video },
+                        });
+                    }}
+                  ></ThumbUpIcon>
+                )}
+
+                {isVideoInPlaylist("watchLater", videoId) ? (
```

Comment:
just a small suggestion here, you can make the variable `isVideoInPlaylist` a boolean by returning a boolean value from the function